crypto/rand/rand_lcl.h

```diff
@@ -117,6 +117,10 @@ struct rand_drbg_st {
     RAND_DRBG *parent;
     int secure; /* 1: allocated on the secure heap, 0: otherwise */
     int nid; /* the underlying algorithm */
+    /*
+     * The value of the rand_fork_count global as of when we last reseeded.
+     * Used to provide fork-safety and reseed this DRBG in the child.
+     */
```

Comment:
Just a suggestion for a minor enhancement. Not a must-do.
``` 
/*
 * Stores the value of the rand_fork_count global as of when we last reseeded.
 * The DRG reseeds automatically whenever drbg->fork_count != rand_fork_count.
 * Used to provide fork-safety and reseed this DRBG in the child process.
 */
```

Reply:
I would rather see that enhancement.